src/sharepoint/sharepoint.service.js

```diff
@@ -95,6 +95,17 @@
           });
       }
 
+      /**
+       * Is a sharepoint migrated to the new billing platform?
+       * @param {string} serviceName
+       */
+      isSharepointMigrated(serviceName) {
+        return this.OvhHttp
```

Comment:
Since you've added the `ovh-api-services` as a peerDependency, could you please refer to: https://github.com/ovh-ux/ovh-api-services/blob/84f75fcee788e4e786cefdf55fe3012d7ffda94b/src/msServices/sharepoint/sharepoint.v6.service.js#L19

That may help you to get rid off `OvhHttp` service in that case.

Reply:
Related to https://github.com/ovh-ux/ovh-module-sharepoint/pull/91/files#diff-779ae1e91249c0bbac462e311817ef08R16, it seems that this method is never used.

Reply:
Good catch, I forgot to remove that :)